Makefile

```diff
@@ -82,9 +82,7 @@ generate-version-file: ## Generates the app version file
 
 .PHONY: test
 test: ## Run tests
-	flake8 .
-	isort --check-only ./app ./tests
-	black --check .
```

Comment:
did we intend to remove `black` here?

Reply:
No :) (I initially thought ruff could replace black but it doesn't - [yet](https://github.com/charliermarsh/ruff/issues/1904))